Use htslib-release bundle instead of direct GIT clone

landesfeind opened the pull request:
Hi Pierre,
thank you for creating the package - I hope it will prove useful as well ;-)

I had to modify the install script a bit to make it run properly in my conda-based R-environment where I am used dedicated compiler and include directories. 
In such environments, the installation only works if the configure-script is executed which is available in the bundled release.

Best,
Manuel

landesfeind committed:
Use htslib-release bundle instead of direct GIT clone

The bundled release contains the configure script allowing for proper
configuration in non-standard system (e.g., conda environments).

lindenb commented:
@landesfeind thank you for the 1st contribution !  but it looks like your modification broke the installation: https://travis-ci.org/github/lindenb/rbcf/builds/680011242#L975 .

i'll be happy to merge it it works with conda but please fix this before :-)

landesfeind committed:
Ensure building htslib with -fPIC

When configure is called, it overwrote the previous attachment of -fPIC. New order
of commands ensures that -fPIC is added as a CFLAG parameter

landesfeind committed:
Change the -fPIC in config.mk instead of Makefile

landesfeind commented:
Hi @lindenb, thanks for the feedback. I modified the installation to ensure the -fPIC flag is forwarded to the compiler. :-)

lindenb commented:
:+1:
